modules/pam_sepermit/tst-pam_sepermit-retval.c

```diff
@@ -0,0 +1,71 @@
+/*
+ * Check pam_sepermit return values.
+ */
+
+#include "test_assert.h"
+
+#include <limits.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+#include <security/pam_appl.h>
+
+#define MODULE_NAME "pam_sepermit"
+#define TEST_NAME "tst-" MODULE_NAME "-retval"
+
+static const char service_file[] = TEST_NAME ".service";
+static struct pam_conv conv;
+
+int
+main(void)
+{
+  pam_handle_t *pamh = NULL;
+  FILE *fp;
+  char cwd[PATH_MAX];
+  char *config_filename;
+
+  ASSERT_NE(NULL, getcwd(cwd, sizeof(cwd)));
+
+  ASSERT_LT(0,asprintf(&config_filename, "%s/my_sepermit.conf", cwd));
```

Comment:
Why do you want to have a dynamically allocated `config_filename` instead of a static string like `service_file`?